layouts/shortcodes/blocks/case-studies.html

```diff
@@ -8,16 +8,24 @@
     <h3  style="text-align: center"><a href="/case-studies/" style="color: #3371E3; font-weight: 400">{{ $caseStudiesSection.LinkTitle }}</a></h3>
     <div id="caseStudiesWrapper">
       {{ range $caseStudiesPages }}
-			{{ $logo := .Resources.GetMatch "**{feature,logo}*.svg" }}
+			{{ $dark := .Resources.GetMatch "**{dark_logo}*.svg" }}
+			{{ $logo := .Resources.GetMatch "**{featured_logo}*.svg" }}
+			
 			{{ if not $logo }}
 				{{ $logo = .Resources.GetMatch "**logo*.png" }}
 			{{ end }}
-      <div>
-        {{ with $logo }}<img src="{{ .RelPermalink }}" alt="{{ .Title }}">{{ end }}
+			{{ if not $dark }}
+				{{ $dark =  $logo }}
+			{{ end }}
+	  <div>
+        <picture>
+	  	  <source srcset={{$dark.RelPermalink}} media="(prefers-color-scheme: dark)">
+	  	    <img src="{{ $logo.RelPermalink }}" alt="{{ .Title }}">
+		</picture>
```

Comment:
I'd like to see a srcset for light backgrounds as well as for dark backgrounds. As much as possible, let's not assume that https://k8s.io/ and the case studies page default to a light background; one day, they may not.

Reply:
I'm wary that the follow up work might not get done, and that we do end up with an implicit light-over-dark preference.

I suggest:
- ensure all the logos do work right on dark and light backgrounds, including updates per existing feedback
- let me know that's done
- I'll do this bit of the work and let you know the Git commands to update your fork to include that work.